data_model.md

```diff
@@ -226,10 +226,24 @@ for more detailed explanation of forecast parameters):
 Additional parameters, such as model configuration parameters, may be
 specified under the “extra parameters” key.
 
-
 Each Forecast has data associated with it. The format of this data is
 found in the Data Format section below
 
+#### Probabilistic forecasts
+{: .anchor}
+
+The Solar Forecast Arbiter supports the specification of
+probabilistic forecasts in terms of a cumulative distribution function
+(CDF). The metadata for a probabilistic forecast container is the same
+as for a [forecast](#forecasts) with the
+addition of two attributes:
+
+-   *Axis* - The axis on which the CDF is specified. *x* corresponds to
+    fixed variable values. *y* corresponds to fixed quantiles.
```

Comment:
`percentiles` instead of `quantiles`, here and elsewhere.  `quantiles` implies an equal-probability spacing, e.g., 0.0, 0.25, 0.5, etc.  0.05, 0.25, 0.50, etc. are not quantiles in the formal sense.